src/virtualmachinesInstance/README.md

```diff
@@ -0,0 +1,7 @@
+# VirtualMAchineIntance
```

Comment:
please change to `VirtualMachineInstance` :)
I also think we should change the directory name to `virtualmachineinstance` for naming consistency

Reply:
should be the kink ... will look what the kind is